packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/middleware/logging.ts

```diff
@@ -25,9 +26,9 @@ export const incomingRequestLogger = (
 
 export const outgoingLog = (_request: Request, response: Response) => {
   // in milliseconds
-  const responseTime = new Date().getTime() - response.locals.requestStartTime
+  const requestTime = new Date().getTime() - response.locals.requestStartTime
   const statusCode = response.statusCode
-  response.locals.logger.info({ responseTime, statusCode }, 'request completed')
+  response.locals.logger.info({ requestTime, statusCode }, 'request completed')
```

Comment:
Since this is measuring how long the request takes (not how long to get a response), I renamed this var